content/doc/book/installing/_kubernetes.adoc

```diff
@@ -552,3 +553,183 @@ This may also be found at:
 ----
 
 You have successfully installed Jenkins on your Kubernetes cluster and can use it to create new and efficient development pipelines.
+
+==== Install Jenkins with Jenkins Operator
+
+Jenkins operator is a Kubernetes native operator who manages operations for 
+Jenkins on Kubernetes. 
+It was built with immutability and declarative configuration as code in mind. 
+The Jenkins Operator is easy to install with just a few manifest and allows 
+users to configure and manage Jenkins on Kubernetes.
+
+===== Prerequisites
+
+Jenkins Operator::
+If you don't have Jenkins Operator installed and configured locally, 
+see the sections below to <<Install Jenkins Operator>>.
+
+===== Install Jenkins Operator
+
+Requirements::
+
+To run Jenkins Operator, you will need:
+
+. Access to a Kubernetes cluster. If you don't have a running Kubernetes cluster, 
+see the link:/doc/book/installing/kubernetes/#create-a-kubernetes-cluster-with-minikube[Create a Kubernetes cluster with minikube] section above.
+
+. kubectl version 1.11+
+
+===== Configure Custom Resource Definition
+
+Custom resources definition (CRD) was introduced in Kubernetes 1.7 and it 
```

Comment:
The Custom Resource Definition (CRD) API has been introduced to Kubernetes in v1.7 and it